### ISSUE TITLE
Rendered diagrams are missing text

### ISSUE THREAD

**Issue by joshuacurtiss** [azat-io/vuepress-plugin-mermaid-wrapper#3]
Hello, thank you for your plugin. I'm trying to test it, and on a plain starter vuepress site, my diagrams will render but there are no labels/text. 

Proc: arm64
Node: v20.12.2

package.json
```json
{
  "name": "vuepress-test",
  "scripts": {
    "docs:dev": "vuepress dev docs",
    "docs:build": "vuepress build docs"
  },
  "devDependencies": {
    "@vuepress/bundler-vite": "2.0.0-rc.18",
    "@vuepress/bundler-webpack": "2.0.0-rc.18",
    "@vuepress/theme-default": "2.0.0-rc.60",
    "sass-embedded": "1.81.0",
    "sass-loader": "16.0.3",
    "vuepress": "2.0.0-rc.18"
    "vuepress-plugin-mermaid-wrapper": "1.15.0"
  }
}
```

config.js
```js
import { webpackBundler } from '@vuepress/bundler-webpack'
import { defaultTheme } from '@vuepress/theme-default'
import { defineUserConfig } from 'vuepress'
import { mermaidWrapperPlugin } from 'vuepress-plugin-mermaid-wrapper';

export default defineUserConfig({
  bundler: webpackBundler(),
  theme: defaultTheme(),
  plugins: [ mermaidWrapperPlugin({}) ],
});
```

README.md
````md
# Hello VuePress

```mermaid
  graph TD
    a[Apple] -->|Text| b[Boy]
```
````

Result:
![Screenshot 2024-11-23 at 3 21 47 PM](https://github.com/user-attachments/assets/745407c4-c232-4448-a48f-964bc2c7794c)

What am I doing wrong? How to fix? Thanks for your help!

**Comment by azat-io** [comment]
That looks weird. Do you have a repository where the problem can be reproduced?

**Comment by adelinn** [comment]
Happened to me too.
I'm using 
```json
    "@vuepress/bundler-vite": "^2.0.0-rc.18",
    "vuepress": "^2.0.0-rc.18",
    "vuepress-plugin-mermaid-wrapper": "^1.15.0"
```

**Comment by adelinn** [comment]
Researched a bit more and seems it's because of some DOMPurify sanitization that happens inside Mermaid library.
Dompurify removes all tags within <foreignObject> tags, so it removes all text.

**Comment by adelinn** [comment]
I just updated the mermaid library in the node_modules of this package and it started working properly. Could you update that so that things start working again?

**Comment by adelinn** [comment]
Linking #4 here
The PR updates mermaid package version which should fix this.

**Comment by adelinn** [comment]
@azat-io when do you expect to release the version that includes this fix?

**Comment by azat-io** [comment]
Released in [v1.16.0](https://github.com/azat-io/vuepress-plugin-mermaid-wrapper/releases/tag/v1.16.0).

**Comment by adelinn** [comment]
Great. Thanks.
